packages/ember-engines/addon-test-support/index.js

```diff
@@ -25,7 +26,7 @@ export async function setupEngine(hooks, engineName) {
 }
 
 function ownerHasEngine(owner, engineName) {
-  return Boolean(owner.factoryFor(`engine:${engineName}`));
+  return owner.hasRegistration(`engine:${engineName}`);
```

Comment:
Why this change is necessary?

Reply:
@villander this doing exactly the same thing, except with `factoryFor` it's two step: you get a factory class and then cast to `boolean` via `Boolean` function.

with `hasRegistration` you get boolean right away

Reply:
Thanks!